options.go

```diff
@@ -177,6 +183,25 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// Installs RingChecksumStatPeriod, which determines the period between
```

Comment:
Yeah, I want to. And add it to Travis too. First step is to add a `make lint`...